packages/bierzo-wallet/src/routes/account/index.ts

```diff
@@ -0,0 +1,5 @@
+import AccountManage from "./manage";
+import AccountRegister from "./register";
+import AccountUpdate from "./update";
+
```

Comment:
Sorry, do not understand you.

Reply:
You have the subdirectories: manage, register, and update. All of them use the `AccountEntity` type and `AccountProps` interface from the register subdirectory. Since the three of them use them, it no longer makes sense to have those declared in register, they should be declared on the root, which is this file.

So between the imports and the export you can declare:
```typescript
type AccountEntity = "iovname" | "starname" | "name";

interface AccountProps {
  entity: AccountEntity;
}
```
Add them to the export, and removed them from the register's index.tsx.